targets/unity-v2/Testing/Tests/Client/ClientApiTests.cs

```diff
@@ -254,14 +257,31 @@ private void GetUserDataCallback2(GetUserDataResult result)
                 int.TryParse(userDataRecord.Value, out actualValue);
             testContext.IntEquals(_testInteger, actualValue);
             testContext.NotNull(userDataRecord, "UserData record not found");
+            testContext.EndTest(UUnitFinishState.PASSED, actualValue.ToString());
+        }
+
+        /// <summary>
+        /// CLIENT API
+        /// Test that the DateTime fields deserialize correctly, and accurately represent UTC time
+        /// Relies on the local machine time and time zone being set correctly.
```

Comment:
What does it mean to set the machine correctly? How will this work on our VMs? (I'm assuming we expect them to default to the time zone based on the data center?)

Reply:
Basically to have a clock that is accurately synced to internet-time, within the given threshold, of 5 minutes. In my experience, if you look at a dozen or more machines, one machine can deviate from an authoritative clock by upto ~2 minutes, even if it's set "properly".